src/actions/GridAlign.js

```diff
@@ -65,6 +65,27 @@ var GridAlign = function (items, options)
             //  We keep laying them out vertically until we've done them all
             tempZone.y += cellHeight;
         }
+        else if (heightSet && !widthSet)
+        {
+            console.log('heighta göre set!');
```

Comment:
This line needs removing, please.

Reply:
I've just removed that console 👍  

Was testing out with that :)